src/Sulu/Component/Content/Document/Behavior/AuditableBehavior.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Behavior;
+
+use Sulu\Component\DocumentManager\Behavior\Audit\TimestampBehavior;
+
+/**
+ * This behavior combines the creator and changer with the created and changed dates.
+ */
+interface AuditableBehavior extends BlameBehavior, TimestampBehavior
```

Comment:
Why is this interface name suffixed with "Behavior"? Interfaces should be named after their role, like "Auditable", "CanBeAudited", "SupportsAuditing" or the like.